### DISCUSSION TITLE
怎么让游客可以直接根据http://***/test/test.txt直接下载，但是确看不到文件

### DISCUSSION THREAD

**Discussion by newgamevip** [AlistGo/alist#3960] [General]
有没有大佬有群拉一个也行啊

**Comment by Rektyfikowany** [comment]
关闭`签名所有`，`元信息`中设置隐藏，就可以用链接下载，
不知道是不是想问这个

**Reply by newgamevip** [reply]
关闭签名所有我不知道在哪，但我感觉元信息设置成全部隐藏应该可以。百度了几个正则表达式都不对，可以写个吗大佬

**Reply by Rektyfikowany** [reply]
阅读[全局配置](https://alist.nn.ci/zh/config/global.html)和[正则表达式](https://developer.mozilla.org/zh-CN/docs/Web/JavaScript/Guide/Regular_Expressions)有关文档，
因管理员可以看到隐藏，设置好退出后才会看到元信息的隐藏效果

**Reply by newgamevip** [reply]
签名所有关了，但是元信息的隐藏表达式总是不行   
可以屏蔽掉字母数字，但是中文的这个带\u的就不行
[\u4e00-\u9fa5]
错误error parsing regexp: invalid escape sequence: `\u` 
大佬指点下

**Reply by Rektyfikowany** [reply]
Go的正则可以用`[\x{4e00}-\x{9fa5}]`，不过推荐`[\p{Han}]`匹配中文

**Reply by newgamevip** [reply]
可以了谢谢大佬。
我有一个g的文件在在liunx系统服务器上，但是从服务器下载到本地由于带宽问题才400多kb很慢。
如果我用curl  -X PUT“地址” 传到百度云盘上会快一些吗？

**Reply by Rektyfikowany** [reply]
速度看你上传的主机到百度云之间的带宽，从服务器直接上传一般更快一些，具体取决于你的网络环境了

**Reply by newgamevip** [reply]
好的，谢谢解答
